schema.sql

```diff
@@ -0,0 +1,9 @@
+ CREATE DATABASE vet_clinic;
+\c vet_clinic;
+ CREATE TABLE animals(
+ id INT,
```

Comment:
[OPTIONAL] Good work creating an animals table 💯 

[OPTIONAL] You could add to the `id` column, the following properties to help you while creating animals. 

`id int NOT NULL IDENTITY(1,1)`